include/rfb-proto.h

```diff
@@ -123,6 +125,21 @@ enum rfb_led_state {
 	RFB_LED_STATE_CAPS_LOCK = 1 << 2,
 };
 
+enum rfb_ext_clipboard_flags {
+	RFB_EXT_CLIPBOARD_FORMAT_TEXT = 1 << 0,
+	RFB_EXT_CLIPBOARD_FORMAT_RTF = 1 << 1,
+	RFB_EXT_CLIPBOARD_FORMAT_HTML = 1 << 2,
+	RFB_EXT_CLIPBOARD_FORMAT_DIB = 1 << 3,
+	RFB_EXT_CLIPBOARD_FORMAT_FILES = 1 << 4,
+	RFB_EXT_CLIPBOARD_CAPS = 1 << 24,
+	RFB_EXT_CLIPBOARD_ACTION_REQUEST = 1 << 25,
+	RFB_EXT_CLIPBOARD_ACTION_PEEK = 1 << 26,
+	RFB_EXT_CLIPBOARD_ACTION_NOTIFY = 1 << 27,
+	RFB_EXT_CLIPBOARD_ACTION_PROVIDE = 1 << 28,
+	// Above known actions
+	RFB_EXT_CLIPBOARD_ACTION_ALL = 0x1e000000,
```

Comment:
Suggestion: If this is defined by ORing together the actions, then the code is self-documenting and doesn't need the comment.